### PR TITLE
Fixes for parts of Keto documentation

### PR DESCRIPTION
**Pull request by kevgo**
I tried to keep it to simple corrections without changing too much. Sometimes I saw an opportunity to move content around in order to say things in a more intuitive order or avoid saying the same thing twice in different places.

### CODE REVIEW

File: docs/keto/configuration.md [hunk 1]
```diff
@@ -3,82 +3,92 @@ id: configuration
 title: Configuration
 ---
 
-```yaml
 # ORY Keto Configuration
-#
-#
+
+You can configure ORY Keto via a config file or environment variables.
```
Comment (kevgo) [inline docs/keto/configuration.md]: This was all in comments in the config file. I have extracted the parts that don't seem related to configuring Keto, like how to set up env variables or Docker, into Markdown documentation on top of this file.

File: docs/keto/engines/acl.md [hunk 2]
```diff
@@ -9,16 +9,16 @@ is a matrix of users and permissions:
 
 |       | blog_post.create | blog_post.delete | blog_post.modify | blog_post.read |
 | ----- | ---------------- | ---------------- | ---------------- | -------------- |
-| alice | yes              | yes              | yes              | yes            |
-| bob   | no               | no               | no               | yes            |
-| peter | yes              | no               | yes              | yes            |
+| Alice | yes              | yes              | yes              | yes            |
```
Comment (kevgo) [inline docs/keto/engines/acl.md]: One liberty I took is capitalizing people's names, at least in text. In code, they could still be lowercase since it's data. But in Markdown it reads really weird if they are lowercase.

File: docs/keto/engines/acp-ory.md [hunk 3]
```diff
@@ -547,25 +526,23 @@ developed over the years at ORY.
 
 ### URNs
 
-> “There are only two hard things in Computer Science: cache invalidation and
-> naming things.” -- Phil Karlton
+> “There are only two hard things in Computer Science: cache invalidation,
+> naming things, and off-by-one errors.” -- Phil Karlton
```
Comment (kevgo) [inline docs/keto/engines/acp-ory.md]: Added this joke, not sure if it is from Phil Karlton. But I think it adds a little breath of fresh air while reading through this long piece of security documentation.

File: docs/keto/engines/index.md [hunk 4]
```diff
@@ -1,24 +1,18 @@
 ---
 id: index
-title: Introduction
+title: Access Controle Engines - Introduction
```
Comment (kevgo) [inline docs/keto/engines/index.md]: This might be too long to render properly. But just "introduction" seems not enough info here.

**Comment by kevgo** [comment]
Sorry for the long PR. This has accumulated over several plane flights without Wifi. I'll send additional changes in a separate branch.

File: docs/keto/configuration.md (hunk 1 above)
Comment (aeneasr) [reply docs/keto/configuration.md]: This is being auto-generated from [here](https://github.com/ory/keto/blob/master/docs/config.yaml) and thus overwritten on every git push to master. The script doing that is this one: https://github.com/ory/keto/blob/master/scripts/run-configuration.sh

Not a merge blocker, but changes will be lost on next push to keto.

File: docs/keto/engines/acp-ory.md (hunk 3 above)
Comment (aeneasr) [reply docs/keto/engines/acp-ory.md]: :)
